09week/hackernews/script.js

```diff
@@ -1 +1,43 @@
 'use strict';
+
+'use strict';
+class Hacker extends React.Component{
+  constructor(props){
+    super(props);
+
+    this.state = {
+      item : []
+    }
+  }
+  componentWillMount(){
+    const stories = [];
+    for(let i=1; i < 111; i++){
```

Comment:
This is fetching the first 110 articles in the history of the site, instead of the recent articles.
To get the recent articles you should use ```https://hacker-news.firebaseio.com/v0/topstories.json?print=pretty```